src/main/kotlin/DetektDemo.kt

```diff
@@ -1 +1,51 @@
-class DetektDemo {}
+class DetektDemo {
+    private   val unusedVariable :   String = "unused"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/b1db8249b3b7765813bc2400c54d7c55f4c41ab9/src/main/kotlin/DetektDemo.kt#L2"></span>Private property unusedVariable is unused.</td>
    </tr>
  </tbody>
</table>